coverage/.resultset.json

```diff
@@ -0,0 +1,702 @@
+{
+  "RSpec": {
+    "coverage": {
+      "/Users/meighanrasley/ada/project_forks/MediaRanker/spec/controllers/albums_controller_spec.rb": [
+        1,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
```

Comment:
1, 1, 1, 1, null, null, null, 1, 1, 1, null, null.... mmmmm..... This file is not very meaningful to look at. You might want to add the coverage folder to your .gitignore next time.